script/download-fonts

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+destination="`pwd`/styles/output/downloaded-fonts"
+# all_file="$destination/_all.css"
+
+run() {
+    destination_css="${destination}/$1.css"
+    if [[ -f "$destination_css" ]]
+    then
+        echo "Skipping $1 because file already exists"
+    else
+        ./script/download-single-font --destination "$destination" --name "$1" "$2"
+    fi
+
+    # echo "@import url('./$1.css');" >> "$all_file"
+}
+
+
+# [[ -f "$all_file" ]] && rm "$all_file"
+[[ ! -d "$destination" ]] && mkdir -p "$destination"
+
+# echo "/* This file and other files in this directory are autogenerated. */" > "$all_file"
+
+run "Roboto-Condensed"    'https://fonts.googleapis.com/css2?family=Roboto+Condensed:wght@400;700&display=swap&subset=latin,latin-ext'
```

Comment:
These should be fixed to be comprehensive and the weights and subsets should be combined